server/game/cards/13.3-PoS/MaesterMullin.js

```diff
@@ -0,0 +1,47 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class MeasterMullin extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel to choose a character',
+            when: {
```

Comment:
Don't need `when` key for actions.

Reply:
Oops, giving away that I just copy/paste vaguely similar cards here :)